policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/senior_or_disabled/assets/limit/couple.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   breakdown:
```

Comment:
could you add a period and remove quotation marks around the label while you are here